commands/command_fsck.go

```diff
@@ -47,7 +47,7 @@ func fsckCommand(cmd *cobra.Command, args []string) {
 		}
 	})
 
-	if err := gitscanner.ScanRefWithDeleted(ref.Sha, nil); err != nil {
+	if err := gitscanner.ScanRef(ref.Sha, nil); err != nil {
```

Comment:
Ah, thanks for linking that -- I forgot about that issue. I agree that the fsck changes could come in a separate pull request, since it seems there are more options re: different ways that that could work.

How do you feel about backing out of this change for now and moving it into a different PR?